test/helpers/SessionAnswersHelperSpec.scala

```diff
@@ -1146,4 +1156,53 @@ class SessionAnswersHelperSpec extends SpecBase {
       }
     }
   }
+
+  "getPreviousUploadsFileNames" should{
+    "return Future[String] " in {
+      val mockRepository = mock(classOf[UploadJourneyRepository])
+      implicit val ec: ExecutionContext = injector.instanceOf[ExecutionContext]
+      val fakeRequestForOtherJourney: UserRequest[AnyContent] = fakeRequestConverter(fakeRequestWithCorrectKeys.withSession(
+        SessionKeys.reasonableExcuse -> "other",
+        SessionKeys.hasConfirmedDeclaration -> "true",
+        SessionKeys.whyReturnSubmittedLate -> "This is a reason.",
+        SessionKeys.whenDidBecomeUnable -> "2022-01-02",
+        SessionKeys.journeyId -> "4321"
+      ))
+      val callBackModel: UploadJourney = UploadJourney(
+        reference = "ref1",
+        fileStatus = UploadStatusEnum.READY,
+        downloadUrl = Some("download.file/url"),
+        uploadDetails = Some(UploadDetails(
+          fileName = "file1.txt",
+          fileMimeType = "text/plain",
+          uploadTimestamp = LocalDateTime.of(2018, 1, 1, 1, 1),
+          checksum = "check1234",
+          size = 2
+        ))
+      )
+      when(mockRepository.getUploadsForJourney(Some("4321"))).thenReturn(Future(Option(Seq(callBackModel))))
+      val  helper = new SessionAnswersHelper(mockRepository)
+      whenReady(helper.getPreviousUploadsFileNames()(fakeRequestForOtherJourney)){
```

Comment:
Understandably whenReady does the same job as await but I prefer we keep it consistent and use 'await'
```suggestion
      await(helper.getPreviousUploadsFileNames()(fakeRequestForOtherJourney)) shouldBe "file1.txt"
```

Reply:
with 'await' there's a compilation error  - could not find implicit value for parameter timeout: akka.util.Timeout

Reply:
The implicit parameter can be found in play.api.Helpers
If you put `import play.api.test.Helpers._` with the other imports, this should fix the problem